include/net/lwm2m_client_utils.h

```diff
@@ -179,6 +181,12 @@ struct lwm2m_fota_update_request {
 	int dfu_type;
 };
 
+/** Event data for id LWM2M_FOTA_UPDATE_MODEM_RECONNECT_REQ. */
```

Comment:
```suggestion
/** Event data for ID LWM2M_FOTA_UPDATE_MODEM_RECONNECT_REQ. */
```
Dunno whether "ID" is needed here at all. Same for the line 190.